src/test/java/task5/SortGenericTest.java

```diff
@@ -0,0 +1,50 @@
+package task5;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SortGenericTest {
+    SortGeneric sortGeneric;
+    Integer[] numbers;
+    String[] strings;
+    Character[] characters;
+    Double[] doubles;
+
+    @BeforeEach
```

Comment:
Wiesz co, zobacz, zmienna number np. jest ci potrzebna tylko w jednym teście, tak samo doubles czy strings. Moim zdaniem raczej nie ma sensu tutaj wsadzać tego do oddzielej metody @BeforeEach init(), bo wtedy przed każdym wywołaniem testu inicjalizuje ci wszystkie zmienne testowe (nawet te których kompletnie nie używać), robi to niepotrzebny spory narzut na pamięć. Przy tych testach raczej tutaj bym zrobił normalnie, w każdym @Test deklaruj sobie zmienną która ci jest potrzebna